conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,16 @@
+-- Products schema
+
+-- !Ups
+
+CREATE TABLE IF NOT EXISTS Product (
+    id          UUID          NOT NULL PRIMARY KEY,
```

Comment:
This will make you (the caller) specify the id upon creation. We don't have to make a change right now, but notice that it will be best practice to let the database create the IDs for us. you can see an example of this in pass-offerings (we use a postgres extension to invoke a function that give us a default value, and leave the id unspecified on row creation). No action on this particular PR

Reply:
No action made on the file, but I did take a look at the pass-offerings evolutions to see `uuid_generate_v4()` called on lines where the id column for a table is declared.